### ISSUE TITLE
[FR] control tag to mimic HTML `<select>` behavior

### ISSUE THREAD

**Issue by addisonklinke** [HumanSignal/label-studio#2906]
**Is your feature request related to a problem? Please describe.**

It's not uncommon for classification labels to have 100s or more options for certain tasks. In this case, the annotator UI is much more concise and readable if the options are placed in a dropdown menu. I've looked through the [documentation](https://labelstud.io/tags/index.html) of available control tags, and am disappointed that this setup doesn't appear to exist

**Describe the solution you'd like**

Add a `Select` control tag which users could use in their labeling setup templates as an alternative to [`Choices`](https://labelstud.io/tags/choices.html)

```xml
<Select name="gender" toName="text" choice="single">
  <Choice value="Man" />
  <Choice value="Woman" />
  <Choice value="Nonbinary" />
  <Choice value="Other" />
</Select>
```

Ideally, the HTML rendering would also have the option to use Bootstrap-select's [live search](https://developer.snapappointments.com/bootstrap-select/examples/#live-search) feature. This would further decrease annotator lead time by allow them to tab to the select element, type a couple characters to filter the results, and use arrow keys + enter to select the desired option

**Describe alternatives you've considered**

1. Existing `Choices` tag: this shows all options at once which makes for a very cluttered labeling UI when there are many choices
2. Wrapping `Choices` with `Filter`: as documented by `@Grzegorz Żurek` in [Slack](https://label-studio.slack.com/archives/C01Q5G6NMKR/p1660920654746599), the filter input doesn't respond to annotators typing but rather continues to display the entire list of choices. In this way, it is no better than the plain `Choices` approach. The same non-responsive behavior is also mentioned [here](https://label-studio.slack.com/archives/CQ8LYPPJS/p1662466845971679)
3. Wrapping with `Taxonomy`: this is the closest to the desired behavior. One problem is there are always "Add" and "Add inside" buttons which allows annotators to define their own option. In many classification tasks, that would need to be disabled so it forces them to pick from the existing options. Community members have also mentioned the JSON export is a bit messy, so I'm hesistant to use this control tag. If `allowAdd` was included as a parameter to `Taxonomy` and the JSON export could be simplified for non-nested taxonomies, I think that would be an acceptable solution

**Additional context**

A third question in [Slack](https://label-studio.slack.com/archives/CQ8LYPPJS/p1661777112029769) asking about this functionality

**Comment by addisonklinke** [comment]
Another shortcoming of `Taxonomy` is that it doesn't support a `perRegion` parameter like `Choices`. That means it can only apply a global classification to an entire image or piece of text (as opposed to a bounding box or word)

**Comment by addisonklinke** [comment]
@makseq How do you feel about re-naming this to focus on improvements to `Taxonomy` vs. creating a new `Select` control tag - perhaps that's easier to maintain? In either case, the desired tag needs to support

1. Searchable dropdown list
2. Tag parameter to enable/disable adding options
3. Support for `perRegion`

**Comment by e-tornike** [comment]
I agree with @addisonklinke, this would be a nice feature, especially the support for `perRegion`.

In addition, `Taxonomy` allows a user to add new labels, which seems is currently not supported by `Labels` or `Choices`. This would be useful for extreme multi-label (classification) tasks (related: #3846).

**Comment by AbubakarSaad** [comment]
Hello,

Taxonomy now supports the `perRegion` tag.
